app/models/verification/outdated_labware/base.rb

```diff
@@ -29,7 +29,7 @@ def labware_from_barcodes(barcodes)
   # @return [Boolean] returns true if the validation, scanning, and audit
   # creation were all successful, and false otherwise.
   def validate_and_create_audits?(params)
-    create_or_get_process_plate(params)
+    @process_plate ||= create_process_plate(params)
```

Comment:
@process_plate does not belong to child. We are not supposed to do that here. Our version was only providing a method to do that and hiding this detail and relying on parent to do the assignment.